services/datamanager/datasync/sync.go

```diff
@@ -41,7 +41,8 @@ const MaxParallelSyncRoutines = 1000
 
```

Comment:
Change MaxParallelSyncRoutines to 500? Or whatever you tested with on an rPi?

Reply:
Chatted offline, decided to keep as 1000 for now as this new implementation will still use less memory than before. This is also easily configurable via config . I tested creating a new syncer with a different number of threads does work.